learningresources/models.py

```diff
@@ -46,10 +46,6 @@ class LearningResource(models.Model):
     xa_avg_grade = models.FloatField(default=0)
     xa_histogram_grade = models.FloatField(default=0)
 
-    class Meta:
```

Comment:
Is this removal intentional? If so I'm fine with it, commenting just in case it's an oversight